src/style.css

```diff
@@ -1,3 +1,87 @@
+@font-face {
+  font-family: Roboto;
+  font-weight: 500;
+  src: url(https://fonts.google.com/specimen/Roboto);
+}
+
+html{
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  background-color: #a09595;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  background-color: #fff;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.logo_image {
+  height: 100%;
+  width: 100%;
+}
+
+.nav_list {
```

Comment:
The class `.nav_list` seems to be used for a link, but it has properties that are typically used for a list (`display: flex; align-items: center;`). This can be confusing. Consider using more descriptive class names.